src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs

```diff
@@ -1,88 +1,70 @@
-﻿using Microsoft.AspNetCore.Components;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
 using Microsoft.AspNetCore.Components.Web;
 
 namespace Bit.Client.Web.BlazorUI
 {
     public partial class BitMessageBar
     {
-        private MessageBarStyle messageBarStyle = MessageBarStyle.Default;
+        protected override string RootElementClass => "bit-msg-bar";
 
-        [Parameter] public RenderFragment? ChildContent { get; set; }
+        [Parameter]
+        public bool IsMultiline { get; set; } = true;
```

Comment:
only parameters with more than one line code can have their [Parameter] attribute in a separate line.
please always consider this convention.

Reply:
Changed all properties according to our convention